packages/gatsby/src/pages/index.js

```diff
@@ -180,10 +173,10 @@ const IndexPage = ({data, searchState, onSearchStateChange}) => {
           <Hero>
             <SectionContent>
               <HeroTitle>
-                Safe, stable, reproducible projects
+        Safe, stable, reproducible projects
```

Comment:
Invalid indent?

Reply:
Oops, forgot that the `@typescript-eslint/indent` autofix doesn't reindent text in JSX. 😅 
Fixed!